docs/manuals/source/Configuration/Director.rst

```diff
@@ -1151,42 +1151,41 @@ The directives within an Options resource may be one of the following:
 .. config:option:: dir/fileset/include/options/HardLinks
 
    :type: yes|no
-   :default: no 
+   :default: no
+
+   .. note::
 
-   New default since :sinceVersion:`23.0.0: Bareos >= 23.0.0`
+      Since :sinceVersion:`23.0.0: fileset keep hard links option switched from
+      yes to :strong:`no`.
```

Comment:
The text between `:` and the closing ` is not printed. It is only used for searching I think.
```suggestion
      In previous versions the default was :strong:`yes`, but since :sinceVersion:`23.0.0: HardLinks = no` the default is :strong:`no`. 
```